target/linux/ath79/dts/qca9531_comfast_cf-e313ac.dts

```diff
@@ -0,0 +1,143 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "comfast,cf-e313ac", "qca,qca9531";
+	model = "COMFAST CF-E313AC";
+
+	aliases {
+		serial0 = &uart;
+		led-boot = &wlan;
+		led-failsafe = &wlan;
+		led-upgrade = &wlan;
+		label-mac-device = &eth0;
```

Comment:
Thanks for adding that already.

However, eth0 would be <&art 0x6> while eth1 is <&art 0x0>, which is at least suspicious.

Are you sure?

eth0 would be:
```
. /lib/functions/system.sh
get_mac_binary "/proc/device-tree/ahb/eth@19000000/mac-address" 0
```

Reply:
Hi Adrian,

Sorry, you're right. I got a bit confused by the fact that &eth0/1 are swapped
```
eth0      Link encap:Ethernet  HWaddr 40:A5:EF:00:00:40  <===== The MAC on the label
          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1
          RX packets:1667 errors:0 dropped:0 overruns:0 frame:0
          TX packets:344 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1000 
          RX bytes:339772 (331.8 KiB)  TX bytes:45870 (44.7 KiB)
          Interrupt:5 

eth1      Link encap:Ethernet  HWaddr 40:A5:EF:2E:45:42  
          inet addr:10.1.24.77  Bcast:10.1.24.95  Mask:255.255.255.224
          inet6 addr: fe80::42a5:efff:fe2e:4542/64 Scope:Link
          inet6 addr: fd00:1714:1714:2032::6cf/128 Scope:Global
          inet6 addr: fd00:1714:1714:2032:42a5:efff:fe2e:4542/64 Scope:Global
          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1
          RX packets:20951 errors:0 dropped:0 overruns:0 frame:0
          TX packets:263 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1000 
          RX bytes:2871476 (2.7 MiB)  TX bytes:23766 (23.2 KiB)
          Interrupt:4
```